src/main/java/core/basesyntax/Record.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Record {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private String originalData;
```

Comment:
this field is redundant. Original data theoretically may differ if we use different sources of input. It's better not to add it to the model